content/applications/sales/subscriptions/renewals.rst

```diff
@@ -28,14 +28,20 @@ with a configured :guilabel:`Recurring Plan` selected, which turns it into a sal
 
 .. note::
   - Only a singular product is required.
-  - A subscription service is still a product (albeit a digital, recurring one)
+  - A subscription service is still a product (albeit a digital, recurring one).
```

Comment:
Instead of digital, I would try to define it using our terms (goods, service, etc).  Unless I am mistaken, I don't think we use the term digital to refer to products? (If I am wrong feel free to ignore this!)